doc/tutorials/calib3d/camera_multiview_calibration/multiview_calibration.markdown

```diff
@@ -0,0 +1,315 @@
+* `--debug_corners` - pass `True` or `False`. If `True` program shows several random images with detected corners for user to manually verify the detection (default is `False`).
+* `--points_json_file` - pass name of JSON file where image and pattern points could be saved after detection. Later this file could be used to run sample code. Default value is '' (nothing is saved).
+* `--find_intrinsics_in_python` - pass `0` or `1`. If `1` then the Python sample automatically calibrates intrinsics parameters and reports reprojection errors. The multiview calibration is done only for extrinsics parameters. This flag aims to separate calibration process and make it easier to debug what goes wrong.
+* `--path_to_save` - path to save results in pickle file
+* `--path_to_visualize` - path to results pickle file needed to run visualization
+* `--visualize` - visualization flag (True or False), if True only runs visualization but path_to_visualize must be provided
+* `--resize_image_detection` - True / False, if True an image will be resized to speed-up corners detection
+
+Alternatively, the Python sample could be run from JSON file that should contain image points, pattern points, and boolean indicator whether a camera is fisheye.
+The example of JSON file is in `opencv_extra/testdata/python/multiview_calibration_data.json` (currently under [pull request](https://github.com/opencv/opencv_extra/pull/1001)). Its format should be dictionary with the following items:
+* `object_points` - list of lists of pattern (object) points (size NUM_POINTS x 3).
+* `image_points` - list of lists of lists of lists of image points (size NUM_CAMERAS x NUM_FRAMES x NUM_POINTS x 2).
+* `image_sizes` - list of tuples (width x height) of image size.
+* `is_fisheye` - list of boolean values (true - fisheye camera, false - otherwise).
+Optionally:
+* `Ks` and `distortions` - intrinsics parameters. If they are provided in JSON file then the proposed method does not estimate intrinsics parameters. `Ks` (intrinsic matrices) is list of lists of lists (NUM_CAMERAS x 3 x 3), `distortions` is list of lists (NUM_CAMERAS x NUM_VALUES) of distortion parameters.
+* `images_names` - list of lists (NUM_CAMERAS x NUM_FRAMES x string) of image filenames for visualization of points after calibration.
+
+```console
+$ python3 multiview_calibration.py --json_file /path/to/json
+```
+
+The description of flags could be found directly by running the sample script with `help` option:
+```console
+python3 multiview_calibration.py --help
+```
+
+The expected output in Linux terminal for `multiview_calibration_images` data (from `opencv_extra/testdata/python/` generated in Blender) should be the following:
+![](images/terminal-demo.png)
+
+The expected output for real-life calibration images in `opencv_extra/testdata/python/real_multiview_calibration_images` is the following:
+![](images/terminal-real-demo.png)
```

Comment:
> terminal-real-demo.png

Its size is >800kb, which is large.
Please try .jpg format for this image.

Reply:
changed to jpg.

Reply:
Looks like jpg file is not added to commits.

Reply:
Changed all images to jpg format

Reply:
I have several warnings in my markdown file, it says that latex math symbols like '\frac', \cdot', and '\choose' are unknown. Do you know how could I fix this?